dataverse/webapi/C#-NETx/BulkOperations/Utility.cs

```diff
@@ -272,8 +288,186 @@ public static async Task DeleteExampleTable(Service service, string tableSchemaN
 
         }
 
+        //TODO: I don't think you need this method
```

Comment:
If you use CreateEntityKeyRequest/CreateEntityKeyResponse and have the EntityKeyId, you can implement RetrieveEntityKeyRequest/RetrieveEntityKeyResponse like the SDK classes and retrieve a Typed EntityKeyMetadata instance to test.